File: workspace/recommend/model/trade_image.py
from sqlalchemy import Column, Integer, String, BigInteger, BINARY, TIMESTAMP
from sqlalchemy.orm import declarative_base

Base = declarative_base()

class TradeImage(Base):
    __tablename__ = 'trade_image'
    trade_image_id = Column(BigInteger, primary_key=True)
    trade_id = Column(BigInteger)
    upload_file_name = Column(String(255))
    saved_file_name = Column(String(255))
    created_date = Column(TIMESTAMP)
    update_date = Column(TIMESTAMP)
    sequence = Column(Integer)

File: workspace/backend/mpick/src/main/java/com/k9c202/mpick/trade/entity/TradeImage.java
package com.k9c202.mpick.trade.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;
import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;

import javax.persistence.*;
import java.sql.Timestamp;

@Getter
@SuperBuilder
@NoArgsConstructor
@Entity
@Table(name = "trade_image")
public class TradeImage {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "trade_image_id")
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "trade_id")
    private Trade trade;

    @Column
    private String saveFileName;

    @Column
    @CreationTimestamp
    private Timestamp createdDate;

    @Column
    @UpdateTimestamp
    private Timestamp updateDate;

    @Column
    private Integer sequence;
}
